src/ontology/fbbt-edit.obo

```diff
@@ -24985,65 +24985,72 @@ relationship: part_of FBbt:00007309 ! embryonic/larval ventral sensory cluster o
 [Term]
 id: FBbt:00002582
 name: abdominal 1 ventral multidendritic neuron vtd2
-def: "." [FBC:Autogenerated]
+def: "Ventral multidendritic neuron vtd2 located in segment A1. It projects in the subesophageal zone (Qian et al., 2018)." [FlyBase:FBrf0238188]
 subset: cur
 xref: VFB:FBbt_00002582
 intersection_of: FBbt:00002581 ! abdominal ventral multidendritic neuron vtd2
 intersection_of: part_of FBbt:00001748 ! larval abdominal segment 1
+relationship: sends_synaptic_output_to FBbt:00051067 ! larval subesophageal zone
 
 [Term]
 id: FBbt:00002583
 name: abdominal 2 ventral multidendritic neuron vtd2
-def: "." [FBC:Autogenerated]
+def: "Ventral multidendritic neuron vtd2 located in segment A2. It projects in the subesophageal zone (Qian et al., 2018)." [FlyBase:FBrf0238188]
 subset: cur
 xref: VFB:FBbt_00002583
 intersection_of: FBbt:00002581 ! abdominal ventral multidendritic neuron vtd2
 intersection_of: part_of FBbt:00001749 ! larval abdominal segment 2
+relationship: sends_synaptic_output_to FBbt:00051067 ! larval subesophageal zone
 
 [Term]
 id: FBbt:00002584
 name: abdominal 3 ventral multidendritic neuron vtd2
-def: "." [FBC:Autogenerated]
+def: "Ventral multidendritic neuron vtd2 located in segment A3. It projects in the subesophageal zone (Qian et al., 2018)." [FlyBase:FBrf0238188]
 subset: cur
 xref: VFB:FBbt_00002584
 intersection_of: FBbt:00002581 ! abdominal ventral multidendritic neuron vtd2
 intersection_of: part_of FBbt:00001750 ! larval abdominal segment 3
+relationship: sends_synaptic_output_to FBbt:00051067 ! larval subesophageal zone
 
 [Term]
 id: FBbt:00002585
 name: abdominal 4 ventral multidendritic neuron vtd2
-def: "." [FBC:Autogenerated]
+def: "Ventral multidendritic neuron vtd2 located in segment A4. It projects near the dorsomedial fascicle (Qian et al., 2018)." [FlyBase:FBrf0238188]
 subset: cur
 xref: VFB:FBbt_00002585
 intersection_of: FBbt:00002581 ! abdominal ventral multidendritic neuron vtd2
 intersection_of: part_of FBbt:00001751 ! larval abdominal segment 4
+relationship: sends_synaptic_output_to FBbt:00001102 ! larval ventral nerve cord
 
 [Term]
 id: FBbt:00002586
 name: abdominal 5 ventral multidendritic neuron vtd2
-def: "." [FBC:Autogenerated]
+def: "Ventral multidendritic neuron vtd2 located in segment A5. It projects near the dorsomedial fascicle (Qian et al., 2018)." [FlyBase:FBrf0238188]
 subset: cur
 xref: VFB:FBbt_00002586
 intersection_of: FBbt:00002581 ! abdominal ventral multidendritic neuron vtd2
 intersection_of: part_of FBbt:00001752 ! larval abdominal segment 5
+relationship: sends_synaptic_output_to FBbt:00001102 ! larval ventral nerve cord
 
 [Term]
 id: FBbt:00002587
 name: abdominal 6 ventral multidendritic neuron vtd2
-def: "." [FBC:Autogenerated]
+def: "Ventral multidendritic neuron vtd2 located in segment A6. It projects in the ventrolateral fascicle (Qian et al., 2018)." [FlyBase:FBrf0238188]
```

Comment:
perhaps clarify by changing to 'It has a projection that terminates in the ventrolateral fascicle'
(same below)